source/host/System.cc

```diff
@@ -12,102 +12,18 @@ namespace Host {
 
 /// @brief The main entry point for the program.
 /// @addr{0x80008EF0}
-int KSystem::main(int argc, char **argv) {
+void KSystem::main(int /* argc */, char** /* argv */) {
```

Comment:
Maybe question for @vabold but I don't think we care to keep argc or argv. The assumption is we would've already handled arguments by this point. But also, this function isn't called from anywhere anymore. Should this be removed for now?